### PR TITLE
Docs/Job

### PR DESCRIPTION
**Pull request by ajesh-cc**
Hi Ben and Noah. Please review the new section added to cover "Jobs" feature in KintoBlock section.

-Ajesh

### CODE REVIEW

File: docs/kintoblocks/jobs.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Jobs created in KintoHub can be configured as `Run Once` or `Repeat Schedule`.
+
+The **Run Once** option can be configured in two ways:
+
+1. **Pre-Deployment**: This Job will be run before the project deployment is started.
+
+2. **Post-Deployment**: This Job will be run after the project deployed is completed.
+
+Pre-Deployment and Post-deployment jobs can be used in cases where a specific set of commands needs to be executed. For e.g. Running a package.json file for hosting a website.
```
Comment (bappr) [inline docs/kintoblocks/jobs.md]: > For e.g. Running a package.json file for hosting a website.

Does not mean a lot I think.

"Running a database migration"

Comment (Disturbing) [reply docs/kintoblocks/jobs.md]: Agreed - this sentence does not make sene.

File: docs/kintoblocks/jobs.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+5. Click on the **Continue** button.
+
+6. Next, specify the details on `Name`, `Internal Name` and `Source Type`.
+
+7. Click on the **Create Job** button to successfully create the KintoBlock.
+
+   ![Create Job](/docs/assets/create-job.png)
+
+## CRON Job or Configuration of Job
```
Comment (Disturbing) [inline docs/kintoblocks/jobs.md]: Configure a Job

File: docs/kintoblocks/jobs.md (hunk 1 above)
Comment (Disturbing) [reply docs/kintoblocks/jobs.md]: or sending a slack message.

Comment (ajesh-cc) [reply docs/kintoblocks/jobs.md]: Done.

File: docs/kintoblocks/jobs.md (hunk 2 above)
Comment (ajesh-cc) [reply docs/kintoblocks/jobs.md]: Done.

**Pull request merged**